frontend/stubs/functions.py

```diff
@@ -171,6 +172,11 @@ def pow(x, y):
     return x ** y
 
 
+def range(x: int) -> List[int]:
```

Comment:
Yes you are right.
It is totally possible to have optional parameters in stubs, I was planning to do it as an enhancement when the current stubs model is working stable and fine. Do you think it is a priority for now?